test-clients/src/main/java/com/hedera/services/bdd/suites/contract/openzeppelin/ERC1155ContractInteractions.java

```diff
@@ -0,0 +1,154 @@
+		return List.of(
+				erc1155()
+		);
+	}
+
+	private HapiApiSpec erc1155() {
+		final var contents = getFileContents(ContractResources.ERC_1155_BYTECODE_PATH);
+		return defaultHapiSpec(CONTRACT_NAME)
+				.given(
+						cryptoCreate(ACCOUNT1),
+						cryptoCreate(ACCOUNT2),
+						cryptoCreate(OPERATIONS_PAYER).balance(ONE_MILLION_HBARS),
+						fileCreate(CONTRACT_FILE_NAME).payingWith(OPERATIONS_PAYER),
+						updateLargeFile(OPERATIONS_PAYER, CONTRACT_FILE_NAME, contents)
+				)
+				.when()
+				.then(
+						contractCreate(CONTRACT_NAME).bytecode(CONTRACT_FILE_NAME).via("contractCreate").payingWith(OPERATIONS_PAYER),
+						getTxnRecord("contractCreate").logged(), // 121618 gas
+						getAccountBalance(OPERATIONS_PAYER).logged(), // started with 1M hbars
+						getAccountInfo(ACCOUNT1).savingSnapshot(ACCOUNT1 + "Info"),
+						getAccountInfo(ACCOUNT2).savingSnapshot(ACCOUNT2 + "Info"),
+						getAccountInfo(OPERATIONS_PAYER).savingSnapshot(OPERATIONS_PAYER + "Info"),
+						withOpContext((spec, log) -> {
+							var accountOneAddress = spec.registry().getAccountInfo(ACCOUNT1 + "Info").getContractAccountID();
+							var accountTwoAddress = spec.registry().getAccountInfo(ACCOUNT2 + "Info").getContractAccountID();
+							var operationsPayerAddress = spec.registry().getAccountInfo(OPERATIONS_PAYER + "Info").getContractAccountID();
+							var ops = new ArrayList<HapiSpecOperation>();
+
+							/* approve for other accounts */
+							var approveCall1 = contractCall(CONTRACT_NAME, ContractResources.ERC_1155_ABI_APPROVE, accountOneAddress, true)
+									.via("acc1ApproveCall").payingWith(OPERATIONS_PAYER)
+									.hasKnownStatus(ResponseCodeEnum.SUCCESS);
+							ops.add(approveCall1);
+							var approveCall2 = contractCall(CONTRACT_NAME, ContractResources.ERC_1155_ABI_APPROVE, accountTwoAddress, true)
+									.via("acc2ApproveCall").payingWith(OPERATIONS_PAYER)
+									.hasKnownStatus(ResponseCodeEnum.SUCCESS);
+							ops.add(approveCall2);
+
+							/* mint to some account */
+							var mintCall = contractCall(CONTRACT_NAME, ContractResources.ERC_1155_ABI_MINT, 0, 10, accountOneAddress)
+									.via("contractMintCall")
+									.payingWith(OPERATIONS_PAYER)
+									.hasKnownStatus(ResponseCodeEnum.SUCCESS);
+							ops.add(mintCall);
+							/* mint to the contract owner */
+							var mintCall2 = contractCall(CONTRACT_NAME, ContractResources.ERC_1155_ABI_MINT, 0, 10, operationsPayerAddress)
+									.via("contractMintCall2")
+									.payingWith(OPERATIONS_PAYER)
+									.hasKnownStatus(ResponseCodeEnum.SUCCESS);
+							ops.add(mintCall2);
+
+							/* transfer from - account to account */
+							var transferCall = contractCall(
+									CONTRACT_NAME,
+									ContractResources.ERC_1155_ABI_SAFE_TRANSFER_FROM,
+									operationsPayerAddress,
+									accountTwoAddress,
+									0, // token id 
+									1, // amount 
+									"0x0"
+							).via("contractTransferFromCall").payingWith(OPERATIONS_PAYER)
```

Comment:
You need to be executing this call fro m`accountOne` and not the operations payer